feedthefox/base/static/css/app.css

```diff
@@ -290,6 +269,12 @@ h3 {
     }
 }
 
+@media only screen and (max-width:992px) {
```

Comment:
Me too, but it overlaps with our many menu items. I'll merge it for now and revisit later.